packages/core/test/unit/html/includePanelProcessor.test.ts

```diff
@@ -112,15 +112,28 @@ test('includeFile import footnote from hash', async () => {
 
   const nodeProcessor = getNewDefaultNodeProcessor();
   const result = await nodeProcessor.process(indexPath, index);
-  // to make lint happy; if write as array.join, some lines more than 100 characters;
-  const a = '<div>\n<p>text<trigger for="pop:footnotefn-1-1"><sup class="footnote-ref">';
-  const b = '<a aria-describedby="footnote-label" href="#fn-1-1">[1]</a></sup></trigger></p>';
-  const c = '</div><hr class="footnotes-sep">\n<section class="footnotes">\n';
-  const d = '<ol class="footnotes-list">\n<popover id="pop:footnotefn-1-1">\n';
-  const e = '            <template #content><div>\n              <p>footnote</p>\n\n';
-  const f = '            </div></template>\n          </popover>\n\n<li id="fn-1-1"';
-  const g = ' class="footnote-item"><p>footnote</p>\n</li>\n\n</ol>\n</section>\n';
-  const expected = a + b + c + d + e + f + g;
+  // to make lint happy
+  let longStr = '<p>text<trigger for="pop:footnotefn-1-1"><sup class="footnote-ref"><a aria-describedby="';
```

Comment:
nit: can we have a better name as opposed to longStr? perhaps something like "footnoteStr" ? 

def think the array.join() is a lot more readable!

and actually i was thinking something like the following!
```typescript
let footnoteStr = 'some text etc ...' 
     + 'more text etc...'
     + 'even more text etc';
```

Reply:
Thanks for clarifying; will be working on that.

Reply:
Fixed in latest commit
<img width="732" alt="Screenshot 2024-03-14 at 16 31 39" src="https://github.com/MarkBind/markbind/assets/121547057/a723d81a-ea4d-416b-abd5-0051d981db88">